subprojects/tooling-api-builders/src/main/java/org/gradle/tooling/internal/provider/runner/TestExecutionBuildConfigurationAction.java

```diff
@@ -109,6 +114,7 @@ private void configureTestTasksForTestDescriptors(Context context) {
         for (final Map.Entry<String, List<InternalJvmTestRequest>> entry : taskAndTests.entrySet()) {
             String testTaskPath = entry.getKey();
             for (AbstractTestTask testTask : queryTestTasks(context, testTaskPath)) {
+                configureTestTask(testTask);
```

Comment:
:thinking: I wonder if we need to run `configureTestTasks` in `applyTasksTo` now, we're re-running configuration in `postProcessExecutionPlan` anyway.

Reply:
That's a good point, I'll check that.

Reply:
You are right, there's no need to call `configureTestTask()` within `applyTasksTo()`.

Reply:
There are fest failures indicating that we cannot simply remove configureTestTask() from here.